### ISSUE TITLE
Why is the effect of pretrained model is so big?

### ISSUE THREAD

**Issue by besaman** [BIT-DA/DCAN#2]
Hi, thanks for sharing the code.
One thing i found is that with using a pretrained model as you do (Scenario of Office31 A->W), I get a 90%+ accuracy for the validation, but surprisingly, when I don't load this pretrained model the accuracy drops to only 4%. Is this usual? and what if I don't want to use the pretrained model ... how to improve the accuracy?
Thanks.

**Comment by BinhuiXie** [comment]
Sincerely sorry for late reply due to my schedule. Our model follows the standard transfer learning procedure of network pre-training and fine-tuning.
